app/src/components/main/modal/darkModalLayout/darkModalLayout.jsx

```diff
@@ -17,9 +17,6 @@
 import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import { CSSTransition } from 'react-transition-group';
```

Comment:
Why are you move up this animation?

Reply:
My whole modal consists of DarkModalLayout and OptionsSection. Without these changes, the animation does not work on OptionsSection.

Reply:
Can the `OptionsSection` be rendered as a child of the `DarkModalLayout`? This seems to fix the issue.